### ISSUE TITLE
FreeBSD encryption lib path error (python sys.platform)

### ISSUE THREAD

**Issue by symphys** [PokemonGoMap/PokemonGo-Map#357]
Running in a FreeBSD jail the following error appears:

```
[MainThread][         utils][   ERROR] Unexpected/unsupported platform 'freebsd10'
Traceback (most recent call last):
  File "runserver.py", line 57, in <module>
    encryption_lib_path = get_encryption_lib_path()
  File "/data/PokemonGo-Map/pogom/utils.py", line 325, in get_encryption_lib_path
    raise Exception(err)
Exception: Unexpected/unsupported platform 'freebsd10'

```

## Expected Behavior
Should work on FreeBSD.

## Possible Solution
utils.py:
Change sys.platform.startswith('freebsd-10'): to elif sys.platform.startswith('freebsd10'):
or even 'freebsd'...

**Comment by jaquer** [comment]
Fixed in 4fd6e9c262bf17bdda4992beeb72d24aa8fd22fb

**Comment by symphys** [comment]
Thanks!
